prototype-starter/app/routes.js

```diff
@@ -1,6 +1,4 @@
-const express = require('express')
-const router = express.Router()
+/* eslint-disable-next-line no-unused-vars */
```

Comment:
is there any way for users to not get these lines, as they probably don't need them and they are a bit daunting

Reply:
Yeah, I think we should look at the starter kit as a whole and look at this line as part of it.